ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -952,7 +954,7 @@
         'Formation: Sharp Turn': '連携：転回斬撃',
         'Formation: Sliding Swipe': '連携：突進斬撃',
         'Four Parts Resolve': '四連断重撃',
-        '(?<!(Maneuver|Firing Order): )High-Powered Laser': '高出力レーザー',
+        '(?<!: )High-Powered Laser': '高出力レーザー',
```

Comment:
BTW, the previous regex caused python error when running `python run_tests.py`: "look-behind requires fixed-width pattern" error, so I change it like this.